src/libstd/fs.rs

```diff
@@ -1814,6 +1814,18 @@ mod tests {
         check!(fs::set_permissions(&out, attr.permissions()));
     }
 
+    #[cfg(windows)]
+    #[test]
+    fn copy_file_preserves_streams() {
```

Comment:
This seems surprising! Could you elaborate on what's going on here? E.g. how come this test is only enabled for Windows?